bioframe/ops.py

```diff
@@ -54,12 +54,19 @@ def select_mask(df, region, cols=None):
     _verify_columns(df, [ck, sk, ek])
 
     chrom, start, end = parse_region(region)
+
     if chrom is None:
         raise ValueError("no chromosome detected, check region input")
-    if (start is not None) and (end is not None):
-        mask = (df[ck] == chrom) & (df[sk] < end) & (df[ek] >= start)
-    else:
+
+    if start is None:
         mask = df[ck] == chrom
+    else:
+        if end is None:
```

Comment:
when is start not None but end is None?

Reply:
Yup, `bioframe.parse_region` now accepts unspecified ends like `"chr2:100-"`